opentech/apply/funds/models/submissions.py

```diff
@@ -309,6 +309,14 @@ class ApplicationSubmission(
     # Workflow inherited from WorkflowHelpers
     status = FSMField(default=INITIAL_STATE, protected=True)
 
+    screening_status = models.ForeignKey(
+        'funds.ScreeningStatus',
+        related_name='+',
+        on_delete=models.CASCADE,
```

Comment:
This should probably be `SET_NULL` or `PROTECT`. We don't want to loose submissions if a status is removed.

Reply:
Of course! Agreed.